content/en/docs/Installation/trustedboot.md

```diff
@@ -243,9 +243,74 @@ install:
   - PARTITION_TWO
 ```
 
-### Notes
+## Notes
 
-#### Mount partitions after install
+### Install additional files from the Installer ISO to the encrypted portions
+
+The ISO installer images can be used to install additional content in the encrypted portion of the disk. 
+
+The osbuilder image can overlay additional files into the iso. For example specify `--overlay-iso /additional/path` to have added the files in the folder inside the ISO. The content can be accessed during installation in `/run/initramfs/live`. 
+For example, to install content of the ISO inside the OEM partition, and to restore those after a reset we can use the following cloud config:
+
+```yaml
+#cloud-config
+
+install:
+  device: "/dev/vda"
+  auto: true
+  partitions:
+    oem:
+      size: 4000
+      fs: ext4
+
+users:
+- name: "kairos"
+  passwd: "kairos"
+
+stages:
+  after-install:
+      - commands:
+        - echo "Copying files to oem and persistent"
+        - /usr/lib/systemd/systemd-cryptsetup attach persistent $(findfs PARTLABEL=persistent) - tpm2-device=auto
```

Comment:
I wonder if we should create a small subcommand to do this (kairos-agent lock/unlock LABEL) so its easier to do it if we expect this workflow to be typical.